src/lib/server/models/discord/attachment/index.ts

```diff
@@ -0,0 +1,17 @@
+import { type InferOutput, nullable, number, object, optional, string } from 'valibot';
+import { Snowflake } from '$lib/server/models/discord/snowflake';
+
+export const Attachment = object({
+    id: optional(Snowflake),
+    filename: string(),
+    title: optional(string()),
+    description: optional(string()),
+    content_type: nullable(optional(string())),
```

Comment:
Is this _actually_ nullable? The docs only note that it is optional, but not nullable.
```suggestion
    content_type: optional(string()),
```

Reply:
Some of the weirdness in `attachment` is actually my attempt to adjust the types to work with my preferred way of implementing the code, but yeah sure I'll prefer the specification/